tests/pages/services/ServiceExternalVolumes-cy.js

```diff
@@ -0,0 +1,153 @@
+require("../../_support/utils/ServicesUtil");
+
+describe("Services", function() {
+  /**
+   * Test the external volumes
+   *
+   * --- IMPORTANT NOTICE ---
+   *
+   * REMEMBER TO UPDATE num_vols_delete IN external-volumes/teardown
+   * WHENEVER YOU ADD OR REMOVE A TEST CASE
+   */
+  describe("External Volumes", function() {
+    beforeEach(function() {
+      cy.configureCluster({
+        jobDetails: true,
+        mesos: "1-for-each-health",
+        nodeHealth: true
+      });
+      cy.visitUrl({
+        url: `services/overview/create`
+      });
+    });
+
+    it("renders proper review screen and JSON for an app with external volume", function() {
+      const serviceName = "app-with-external-volume";
+      const cmdline =
+        "while true ; do echo 'test' > test/echo ; sleep 100 ; done";
+      const volumeName = `dcos-system-test-${Cypress.env("TEST_UUID")}-${serviceName}`;
+
+      // Select 'Single Container'
+      cy.contains("Single Container").click();
+
+      // Fill-in the input elements
+      cy
+        .root()
+        .getFormGroupInputFor("Service ID *")
+        .type(`{selectall}{rightarrow}${serviceName}`);
+
+      cy.root().getFormGroupInputFor("Memory (MiB) *").type("{selectall}64");
+      cy.root().getFormGroupInputFor("Command").type(cmdline);
+
+      // Select Universal Container Runtime (UCR)
+      cy.contains("More Settings").click();
+      cy.get("label").contains("Universal Container Runtime (UCR)").click();
+
+      // Select Volumes section
+      cy.root().get(".menu-tabbed-item").contains("Volumes").click();
+
+      // Add an environment variable
+      cy.contains("Add Volume").click();
+      cy.root().find('select[name="volumes.0.type"]').select("EXTERNAL");
```

Comment:
tests are failing because this is no select anymore, this should already be fixed in the integration tests:
https://github.com/dcos/dcos-ui/commit/20373b72f1cd8663e5769f99ac6004995b97d45c#diff-0796ef68862d3b5d45e239ff56ee5be7R1175